src/nodes/materials/NodeMaterial.js

```diff
@@ -45,6 +45,8 @@ class NodeMaterial extends Material {
 		this.lights = true;
 		this.normals = true;
 
+		this.environment = false;
```

Comment:
The idea is to add `NodeMaterial.environment` that controls which materials support `Scene.environment`/PMREM. `MeshStandardNodeMaterial` is the only material that sets this property to `true`.